### PR TITLE
[RFR] Controls tests modified

### PR DESCRIPTION
**Pull request by kpunwatk**
Signed-off-by: Karishma Punwatkar <kpunwatk@redhat.com>

	modified:   mta/fixtures/apis.py
	modified:   mta/fixtures/business_service.py
	modified:   mta/fixtures/job_function.py
	modified:   mta/fixtures/stakeholder.py
	modified:   mta/fixtures/stakeholder_group.py
	modified:   mta/fixtures/tags.py
	modified:   mta/tests/migration/controls/test_business_services.py
	modified:   mta/tests/migration/controls/test_stakeholder.py
	modified:   mta/tests/migration/controls/test_tags.py

### CODE REVIEW

File: mta/tests/migration/controls/test_stakeholder_group.py
Comment (mguetta1) [inline mta/tests/migration/controls/test_stakeholder_group.py:1]: @kpunwatk please create a separate PR for adding new tests

File: mta/tests/migration/controls/test_job_function.py
Comment (mguetta1) [inline mta/tests/migration/controls/test_job_function.py:1]: @kpunwatk please create a separate PR for adding new tests

File: pytest.ini [hunk 1]
```diff
@@ -6,3 +6,5 @@ markers =
     tags: marks tests as tags
     bs: marks tests as business services
     stakeholder: marks tests as stakeholder
+    sg: marks tests as stakeholder group
+    jf: marks tests as job function
```
Comment (mguetta1) [inline pytest.ini]: @kpunwatk please create a separate PR for adding new tests

**Review by mguetta1 — CHANGES_REQUESTED** [review]

**Comment by mguetta1** [comment]
@kpunwatk please run pre-commit checks

File: mta/tests/migration/controls/test_stakeholder_group.py
Comment (kpunwatk) [reply mta/tests/migration/controls/test_stakeholder_group.py:1]: Thanks! Done.

File: mta/tests/migration/controls/test_job_function.py
Comment (kpunwatk) [reply mta/tests/migration/controls/test_job_function.py:1]: Done

File: pytest.ini (hunk 1 above)
Comment (kpunwatk) [reply pytest.ini]: Done

**Review by mguetta1 — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**